capitulo 6/8-animais_estimação.py

```diff
@@ -0,0 +1,9 @@
+cachorro = {'vira_lata':'Raquel','Poodle':'Ana'}
+gato = {'persa':'Chico','Siamês':'Carla'}
+
+pets = (cachorro, gato)
```

Comment:
Mesma coisa que na questão 7.
```suggestion
pets = [cachorro, gato]
```